weed/command/mount_std.go

```diff
@@ -51,61 +53,75 @@ func runMount(cmd *Command, args []string) bool {
 type WFS struct{}
 
 func (WFS) Root() (fs.Node, error) {
-	return Dir{}, nil
+	return &Dir{Path: "/"}, nil
 }
 
 type Dir struct {
-	Path string
-	Id   uint64
+	Id        uint64
+	Path      string
+	DirentMap map[string]*fuse.Dirent
 }
 
-func (dir Dir) Attr(context context.Context, attr *fuse.Attr) error {
-	attr.Inode = 1
+func (dir *Dir) Attr(context context.Context, attr *fuse.Attr) error {
+	attr.Inode = dir.Id
 	attr.Mode = os.ModeDir | 0555
 	return nil
 }
 
-func (dir Dir) Lookup(ctx context.Context, name string) (fs.Node, error) {
-	files_result, e := filer.ListFiles(*mountOptions.filer, dir.Path, name)
-	if e != nil {
-		return nil, fuse.ENOENT
-	}
-	if len(files_result.Files) > 0 {
-		return File{files_result.Files[0].Id, files_result.Files[0].Name}, nil
+func (dir *Dir) Lookup(ctx context.Context, name string) (fs.Node, error) {
+	if dirent, ok := dir.DirentMap[name]; ok {
+		if dirent.Type == fuse.DT_File {
+			return &File{dirent.Inode, dirent.Name}, nil
+		}
+		return &Dir{
+			Id:   dirent.Inode,
+			Path: path.Join(dir.Path, dirent.Name),
+		}, nil
 	}
-	return nil, fmt.Errorf("File Not Found for %s", name)
+	return nil, fuse.ENOENT
```

Comment:
This error will received by fuse package and not be exported. So just follow the rule of this package.